local-volume/bootstrapper/deployment/kubernetes/example-config.yaml

```diff
@@ -1,21 +1,17 @@
 # The config map is used to configure local volume discovery; it is a map from
 # storage class to its mount configuration.
+apiVersion: v1
```

Comment:
Can you also update the other yaml for gce in this directory?

Reply:
done